.env.example

```diff
@@ -23,5 +23,5 @@ DIRECTORY_GRPC_CERT_PATH=$HOME/.config/topaz/certs/grpc-ca.crt
 # ASERTO_TENANT_ID={Your Aserto Tenant ID guid}
 # ASERTO_AUTHORIZER_API_KEY={Your Authorizer API Key}
 # ASERTO_DIRECTORY_API_KEY={You Directory (read-only) API Key}
-# ASERTO_POLICY_INSTANCE_NAME=policy-todo
-# ASERTO_POLICY_INSTANCE_LABEL=policy-todo
+# ASERTO_POLICY_INSTANCE_NAME=todo-rebac
```

Comment:
we don't yet have everything plumbed to todo-rebac.

so in the immediate term this should be ASERTO_POLICY_INSTANCE_NAME=todo